presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java

```diff
@@ -282,6 +282,21 @@ protected void assertQueryFails(Session session, @Language("SQL") String sql, @L
         QueryAssertions.assertQueryFails(queryRunner, session, sql, expectedMessageRegExp);
     }
 
+    protected void assertQueryError(@Language("SQL") String sql, @Language("RegExp") String expectedMessageRegExp)
+    {
+        assertQueryError(queryRunner, getSession(), sql, expectedMessageRegExp);
+    }
+
+    protected void assertQueryError(QueryRunner queryRunner, Session session, @Language("SQL") String sql, @Language("RegExp") String expectedMessageRegExp)
+    {
+        try {
```

Comment:
Why do you have the try block here ?

Does assertQueryError(queryRunner, session, sql, expectedMessageRegExp); not work ?